diff --git a/node_modules/@sap/artifact-management-base-types/src/cp/CapApi.d.ts b/node_modules/@sap/artifact-management-base-types/src/cp/CapApi.d.ts
index 2bd5375..92a7622 100644
--- a/node_modules/@sap/artifact-management-base-types/src/cp/CapApi.d.ts
+++ b/node_modules/@sap/artifact-management-base-types/src/cp/CapApi.d.ts
@@ -1,4 +1,4 @@
-import { ReflectedModel } from '@sap/cds/apis/reflect';
+type ReflectedModel = any;
 import ProjectFileSystem from '../commons/ProjectFileSystem';
 export declare type CDSDefinition = {
     kind: string;
