diff --git a/node_modules/@sap/artifact-management/dist/src/project-api/WorkspaceImpl.js b/node_modules/@sap/artifact-management/dist/src/project-api/WorkspaceImpl.js
index 5690ada..169e037 100644
--- a/node_modules/@sap/artifact-management/dist/src/project-api/WorkspaceImpl.js
+++ b/node_modules/@sap/artifact-management/dist/src/project-api/WorkspaceImpl.js
@@ -4,13 +4,14 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 const ProjectImpl_1 = __importDefault(require("./ProjectImpl"));
-const optionalRequire = require('optional-require')(require);
-const { workspace } = optionalRequire('vscode') || {};
+// const optionalRequire = require('optional-require')(require);
+// const { workspace } = optionalRequire('vscode') || {};
 class WorkspaceImpl {
-    constructor() {
+    constructor(workspace) {
         this.readModuleAsProject = true;
-        this.workspaceFolders = workspace.workspaceFolders;
-        workspace.onDidChangeWorkspaceFolders((event) => {
+        this.workspace = workspace;
+        this.workspaceFolders = this.workspace.workspaceFolders;
+        this.workspace.onDidChangeWorkspaceFolders((event) => {
             if (this.watch !== undefined) {
                 this.watch(event);
             }
@@ -39,7 +40,7 @@ class WorkspaceImpl {
         return uri;
     }
     startWatch() {
-        if (workspace.workspaceFile) {
+        if (this.workspace.workspaceFile) {
             this.watch = (event) => {
                 let folders = [];
                 let eventName = '';
